src/Arcus.WebApi.Unit/Security/Authentication/SharedAccessKeyAuthenticationAttributeTests.cs

```diff
@@ -127,27 +134,176 @@ public async Task AuthorizedRoute_WithSharedAccessKey_ShouldFailWithUnauthorized
             _testServer.AddService<ISecretProvider>(new InMemorySecretProvider((SecretName, secretValue)));
 
             // Act
-            using (HttpResponseMessage response = 
-                await SendAuthorizedHttpRequestWithHeader(HeaderName, new [] { secretValue, $"second header value other then {nameof(secretValue)}" }))
+            using (HttpResponseMessage response =
+                await SendAuthorizedHttpRequest(AuthorizedRouteHeader, headerName: HeaderName, headerValues: new[] { secretValue, $"second header value other then {nameof(secretValue)}" }))
+            {
+                // Assert
+                Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+            }
+        }
+
+        #endregion HeaderValidation
+
+        #region QueryStringValidation
```

Comment:
We don't really use regions, would you mind removing them please?

Reply:
will do